amd_openvx/openvx/ago/ago_internal.h

```diff
@@ -229,10 +229,12 @@ typedef enum {
     ago_kernel_cmd_query_target_support       =  5,
 #if ENABLE_OPENCL
     ago_kernel_cmd_opencl_codegen             =  6,
+#elif ENABLE_HIP
+    ago_kernel_cmd_hip_codegen                =  6,
 #endif
     ago_kernel_cmd_valid_rect_callback        =  7,
 #if ENABLE_HIP
-    ago_kernel_cmd_hip_execute                =  8,
+    ago_kernel_cmd_hip_execute                =  8,     // adding it for HIP kernels to execute without codegen
```

Comment:
same as above, the comment is removed in ToT. please sync your code and accept the ToT.